cmake/modules/patch-dpdk-conf.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# -*- mode:sh; tab-width:4; indent-tabs-mode:nil -*
+
+setconf() {
+    local key=$1
+    local val=$2
+    if grep -q ^$key= ${conf}; then
+        sed -i "s:^$key=.*$:$key=$val:g" ${conf}
+    else
+        echo $key=$val >> ${conf}
+    fi
+}
+
+conf=$1
+shift
+machine=$1
+shift
+
+setconf CONFIG_RTE_MACHINE "${machine}"
```

Comment:
@yuyuyu101 do you think we can have a functional SPDK backend without all these kernel modules?